README.md

```diff
@@ -72,6 +72,18 @@ To run the project, execute the following command:
 - GitHub: [@githubhandle](https://github.com/Innocent259)
 - LinkedIn: [LinkedIn](https://www.linkedin.com/in/innocent-ndahimana-b4b870245/)
 
+ **Author2**
+
```

Comment:
Please update your readme file to be more professional🙏. It is essential to ensure that your readme documentation includes all information that can help a visitor understand how to use your project and contribute to it👍. Your readme file should contain all sections from the microverse readme template. Only the FAQ section is optional. Please make sure your readme template is identical to the microverse readme template and update your table of contents to include all sections of the readme